core/dialog/model/DialogMapping.py

```diff
@@ -1,40 +0,0 @@
-from typing import Callable, Dict
-
-from core.base.model.ProjectAliceObject import ProjectAliceObject
-from core.commons import constants
-
-
-class DialogMapping(ProjectAliceObject):
-
-	def __init__(self, mapping: Dict[str, Callable]):
-		super().__init__(depth=6)
-
-		caller = self.Commons.getFunctionCaller()
-		self._mapping = {f'{caller}:{state}': func for state, func in mapping.items()}
-		self._state = constants.DEFAULT
```

Comment:
I did not place the caller stuff in there again yet, because I was not quite sure what the depth has to be to retrieve it correctly